wrappers/javascript/indy-vdr-nodejs/src/error.ts

```diff
@@ -3,13 +3,13 @@ import type { IndyVdrErrorObject } from '@hyperledger/indy-vdr-shared'
 import { IndyVdrError } from '@hyperledger/indy-vdr-shared'
 
 import { allocateString } from './ffi'
-import { nativeIndyVdr } from './library'
+import { getNativeIndyVdr } from './library'
 
 export const handleError = (code: number) => {
```

Comment:
Seeing this now, I would say it makes more sense to have `handleError` as a private method on the `NodeJSIndyVdr` class. Could you include that in the PR?